requirements/test-mindeps.txt

```diff
@@ -0,0 +1,56 @@
+#
+# This file is autogenerated by pip-compile with Python 3.11
+# by the following command:
+#
+#    pip-compile --output-file=/home/sirosen/dev/sdk-and-cli/sdk/requirements/test-mindeps.txt --resolver=backtracking /home/sirosen/dev/sdk-and-cli/sdk/requirements/test-mindeps.isolated
```

Comment:
Is there to set this up so the command uses relative paths?

Otherwise, many lines in this file (not just this one) will change each time another dev generates this file, because there are many absolute paths listed throughout the file.

Reply:
Oh boy, I hadn't noticed that... I should find a way of doing this which works with relative paths.
I'm sure it's possible -- I'll dig through the `pip-compile` options to see what we've got on hand.

Reply:
With your recommendation to remove the `{toxinidir}` variable, this resolved. Thanks for calling that out.